### PR TITLE
typos, spelling, grammar

### PR DESCRIPTION
**Pull request by brianteeman**
non-exhaustive corrections

will make it easier for translators if the english strings are correct and understable will make it easier for developers if the english comments are correct and understandable

### CODE REVIEW

File: administrator/com_joomgallery/src/Model/CategoryModel.php [hunk 1]
```diff
@@ -1005,7 +1005,7 @@ public function getSiblings($pk, $side, $setError=false)
       $parent_id = intval($this->item->parent_id);
     }
 
-    // Load catgory table
+    // Load caegory table
```
Comment (Elfangor93) [inline administrator/com_joomgallery/src/Model/CategoryModel.php:1008]: I guess this here is still a typo  ;-)

**Review by Elfangor93 — CHANGES_REQUESTED** [review]

**Comment by Elfangor93** [comment]
Thank you @brianteeman for your effort.
This is huge - hanving a good english speaker looking over my spelling.

**Comment by brianteeman** [comment]
It is not 100% complete as there is too much code to review it all but it should cover most issues. 

Note I did not touch any code related "spelling errors"

**Review by Elfangor93 — APPROVED** [review]

**Review by eumel1602 — APPROVED** [review]
it's ok! Thanks Brian!!!

File: administrator/com_joomgallery/src/Service/Uploader/Uploader.php [hunk 2]
```diff
@@ -357,24 +357,23 @@ public function overrideData(&$data): bool
...
           else
           {
-            // Matadata value not available in image
+            // Metadata value not available in image
             if($this->component->getConfig()->get('jg_replaceshowwarning') > 0)
             {
               $this->component->addWarning(Text::sprintf('COM_JOOMGALLERY_SERVICE_WARNING_REPLACE_NO_METADATA', Text::_($source_name)));
-              $this->component->addLog(Text::sprintf('COM_JOOMGALLERY_SERVICE_WARNING_REPLACE_NO_METADATA', Text::_($source_name)), 'warning', 'jerror');
-            }
+              $this->e
```
Comment (Elfangor93) [inline administrator/com_joomgallery/src/Service/Uploader/Uploader.php:371]: Unintentionally deleted lines of code. Will be fixed with PR #318.

**Pull request merged**